rules/windows/registry/registry_set/registry_set_scr_file_executed_by_rundll32.yml

```diff
@@ -0,0 +1,34 @@
+title: Registry Key set after .scr file execution with Rundll32
+id: 40b6e656-4e11-4c0c-8772-c1cc6dae34ce
+description: Detects registry key established after .scr file execution using Rundll32 through desk.cpl
+status: experimental
+date: 2022/05/04
+modified: 2022/05/04
+author: Jose Luis Sanchez Martinez (@Joseliyo_Jstnk)
+references:
+    - https://twitter.com/VakninHai/status/1517027824984547329
+    - https://twitter.com/pabraeken/status/998627081360695297
+    - https://jstnk9.github.io/jstnk9/research/InstallScreenSaver-SCR-files
+logsource:
+    product: windows
+    category: registry_set
+detection:
+    selection:
+        EventType: SetValue
+        Image|endswith: '\rundll32.exe'
+    registry:
+        TargetObject|contains: '\Control Panel\Desktop\SCRNSAVE.EXE'
+        Details|endswith: '.scr'
+    filter:
+        Details|contains: 
+        - 'C:\Windows\System32\'
+        - 'C:\Windows\SysWOW64\'
+        - 'C:\Windows\system32\'
+        - 'C:\Windows\SysWow64\'
```

Comment:
Hi,
shoud be only `C:\Windows\System32` and `C:\Windows\SysWOW64`.
Elk use for text `keyword` type casa-sensitive or `text` type case-insensitive.
You have to use some sigmac option to deal with.
I kown elastic is a pain in the neck.